src/Console/InfectionContainer.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Console;
+
+use Infection\Config\InfectionConfig;
+use Infection\Differ\DiffColorizer;
+use Infection\Differ\Differ;
+use Infection\EventDispatcher\EventDispatcher;
+use Infection\Finder\Locator;
+use Infection\Mutant\MutantCreator;
+use Infection\Mutator\Util\MutatorsGenerator;
+use Infection\Process\Runner\Parallel\ParallelProcessRunner;
+use Infection\TestFramework\Config\TestFrameworkConfigLocator;
+use Infection\TestFramework\Coverage\CachedTestFileDataProvider;
+use Infection\TestFramework\Coverage\CodeCoverageData;
+use Infection\TestFramework\Coverage\TestFileDataProvider;
+use Infection\TestFramework\Factory;
+use Infection\TestFramework\PhpUnit\Adapter\PhpUnitAdapter;
+use Infection\TestFramework\PhpUnit\Config\Path\PathReplacer;
+use Infection\TestFramework\PhpUnit\Config\XmlConfigurationHelper;
+use Infection\TestFramework\PhpUnit\Coverage\PhpUnitTestFileDataProvider;
+use Infection\Utils\TmpDirectoryCreator;
+use Infection\Utils\VersionParser;
+use PhpParser\Lexer;
+use PhpParser\Parser;
+use PhpParser\ParserFactory;
+use PhpParser\PrettyPrinter\Standard;
+use Pimple\Container;
+use SebastianBergmann\Diff\Differ as BaseDiffer;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * InfectionContainer extending Container enables us
```

Comment:
I wonder if we need such phpdoc tbh

Reply:
I believe there was a question, but if that’s too much I’ll remove it